2013-10-18 06:12:26,243 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.sysman.ias.ias.IASIntegration
2013-10-18 06:12:26,666 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.sysman.eml.target.slb.common.SLBIntegration
2013-10-18 06:12:26,805 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.sysman.eml.ssl.intg.SSLIntegration
2013-10-18 06:12:26,964 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.tip.oem.central.domain.ProcessConnectDomainIntg
2013-10-18 06:12:26,965 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.tip.oem.central.instance.ProcessConnectInstanceIntg
2013-10-18 06:12:26,968 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.webdb.admin.em.PortalIntegration
2013-10-18 06:12:26,969 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.webdb.admin.em.SSOIntegration
2013-10-18 06:12:26,970 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.reports.em.RepIntg
2013-10-18 06:12:26,971 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.sysman.ocs.mntr.target.OcsEmailIntegration
2013-10-18 06:12:26,976 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.sysman.ocs.mntr.target.OcsOidIntegration
2013-10-18 06:12:26,977 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.sysman.ocs.mntr.target.OcsOvfIntegration
2013-10-18 06:12:26,978 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.sysman.ocs.mntr.target.OcsWebconfIntegration
2013-10-18 06:12:26,979 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.sysman.ocs.mntr.target.OcsWirelessIntg
2013-10-18 06:12:26,980 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.sysman.ocs.mntr.target.OcsCalGrpIntegration
2013-11-07 01:52:13,361 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.sysman.ias.ias.IASIntegration
2013-11-07 01:52:13,735 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.sysman.eml.target.slb.common.SLBIntegration
2013-11-07 01:52:13,891 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.sysman.eml.ssl.intg.SSLIntegration
2013-11-07 01:52:14,031 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.tip.oem.central.domain.ProcessConnectDomainIntg
2013-11-07 01:52:14,031 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.tip.oem.central.instance.ProcessConnectInstanceIntg
2013-11-07 01:52:14,031 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.webdb.admin.em.PortalIntegration
2013-11-07 01:52:14,031 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.webdb.admin.em.SSOIntegration
2013-11-07 01:52:14,031 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.reports.em.RepIntg
2013-11-07 01:52:14,031 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.sysman.ocs.mntr.target.OcsEmailIntegration
2013-11-07 01:52:14,031 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.sysman.ocs.mntr.target.OcsOidIntegration
2013-11-07 01:52:14,031 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.sysman.ocs.mntr.target.OcsOvfIntegration
2013-11-07 01:52:14,063 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.sysman.ocs.mntr.target.OcsWebconfIntegration
2013-11-07 01:52:14,063 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.sysman.ocs.mntr.target.OcsWirelessIntg
2013-11-07 01:52:14,063 [Orion Launcher] ERROR app.ContextInitializer contextInitialized.271 - Integration Class not found: oracle.sysman.ocs.mntr.target.OcsCalGrpIntegration
2013-11-07 01:52:15,841 [JobWorker 3:Thread-12] ERROR em.jobs executeCommand.159 - UpdateARUTables: Oracle MetaLink credentials are incorrect or missing. Click Patching Setup to set required parameters.
2013-11-07 02:56:26,407 [OmsServiceDriver thread] WARN  jdbc.ConnectionCache _getConnection.303 - Got a fatal exeption when getting a connection; Error code = 17002; Cleaning up cache and retrying
2013-11-07 02:56:26,423 [OmsServiceDriver thread] WARN  jdbc.ConnectionCache _getConnection.306 - Io 异常: The Network Adapter could not establish the connection
java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:111)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:145)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:254)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:386)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:493)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:164)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:34)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:767)
	at oracle.jdbc.pool.OracleDataSource.getPhysicalConnection(OracleDataSource.java:324)
	at oracle.jdbc.pool.OracleConnectionPoolDataSource.getPooledConnection(OracleConnectionPoolDataSource.java:114)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getNewPoolOrXAConnection(OracleConnectionCacheImpl.java:694)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getPooledConnection(OracleConnectionCacheImpl.java:540)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:346)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:324)
	at oracle.sysman.util.jdbc.ConnectionCache._getConnection(ConnectionCache.java:296)
	at oracle.sysman.util.jdbc.ConnectionCache.getUnwrappedConnection(ConnectionCache.java:423)
	at oracle.sysman.emSDK.svc.conn.FGAConnectionCache.getFGAConnection(FGAConnectionCache.java:156)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.getPrivateConnection(ConnectionService.java:605)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.getRepositoryVersionAndMode(ConnectionService.java:368)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.verifyRepositoryEx(ConnectionService.java:395)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.verifyRepository(ConnectionService.java:450)
	at oracle.sysman.emdrep.jobs.Dispatcher.mainTask(Dispatcher.java:1952)
	at oracle.sysman.emdrep.omsservice.OmsServiceDriver.run(OmsServiceDriver.java:164)
2013-11-07 02:56:26,423 [OmsServiceDriver thread] ERROR conn.ConnectionService verifyRepositoryEx.433 - Invalid Connection Pool. ERROR = Io 异常: The Network Adapter could not establish the connection
2013-11-07 02:56:26,423 [OmsServiceDriver thread] ERROR em.jobs mainTask.1956 - encountered error: 
java.sql.SQLException: Io 异常: Connection reset by peer: socket write error
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:111)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:145)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:254)
	at oracle.jdbc.driver.T4CCallableStatement.executeForRows(T4CCallableStatement.java:972)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1252)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:3290)
	at oracle.jdbc.driver.OraclePreparedStatement.execute(OraclePreparedStatement.java:3395)
	at oracle.jdbc.driver.OracleCallableStatement.execute(OracleCallableStatement.java:4222)
	at oracle.sysman.util.jdbc.PreparedStatementWrapper.execute(PreparedStatementWrapper.java:630)
	at oracle.sysman.emdrep.jobs.Dispatcher.getSteps(Dispatcher.java:1585)
	at oracle.sysman.emdrep.jobs.Dispatcher.process(Dispatcher.java:1203)
	at oracle.sysman.emdrep.jobs.Dispatcher.mainTask(Dispatcher.java:1912)
	at oracle.sysman.emdrep.omsservice.OmsServiceDriver.run(OmsServiceDriver.java:164)
2013-11-07 02:56:31,430 [OmsServiceDriver thread] WARN  jdbc.ConnectionCache _getConnection.303 - Got a fatal exeption when getting a connection; Error code = 17002; Cleaning up cache and retrying
2013-11-07 02:56:31,430 [OmsServiceDriver thread] WARN  jdbc.ConnectionCache _getConnection.306 - Io 异常: The Network Adapter could not establish the connection
java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:111)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:145)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:254)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:386)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:493)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:164)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:34)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:767)
	at oracle.jdbc.pool.OracleDataSource.getPhysicalConnection(OracleDataSource.java:324)
	at oracle.jdbc.pool.OracleConnectionPoolDataSource.getPooledConnection(OracleConnectionPoolDataSource.java:114)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getNewPoolOrXAConnection(OracleConnectionCacheImpl.java:694)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getPooledConnection(OracleConnectionCacheImpl.java:540)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:346)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:324)
	at oracle.sysman.util.jdbc.ConnectionCache._getConnection(ConnectionCache.java:296)
	at oracle.sysman.util.jdbc.ConnectionCache.getUnwrappedConnection(ConnectionCache.java:423)
	at oracle.sysman.emSDK.svc.conn.FGAConnectionCache.getFGAConnection(FGAConnectionCache.java:156)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.getSystemConnection(ConnectionService.java:683)
	at oracle.sysman.emdrep.omsservice.OmsServiceDriver.getConnection(OmsServiceDriver.java:88)
	at oracle.sysman.emdrep.jobs.Dispatcher.getConnection(Dispatcher.java:1054)
	at oracle.sysman.emdrep.jobs.Dispatcher.process(Dispatcher.java:1200)
	at oracle.sysman.emdrep.jobs.Dispatcher.mainTask(Dispatcher.java:1912)
	at oracle.sysman.emdrep.omsservice.OmsServiceDriver.run(OmsServiceDriver.java:164)
2013-11-07 02:56:31,430 [OmsServiceDriver thread] WARN  jdbc.ConnectionCache _getConnection.303 - Got a fatal exeption when getting a connection; Error code = 17002; Cleaning up cache and retrying
2013-11-07 02:56:31,430 [OmsServiceDriver thread] WARN  jdbc.ConnectionCache _getConnection.306 - Io 异常: The Network Adapter could not establish the connection
java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:111)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:145)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:254)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:386)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:493)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:164)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:34)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:767)
	at oracle.jdbc.pool.OracleDataSource.getPhysicalConnection(OracleDataSource.java:324)
	at oracle.jdbc.pool.OracleConnectionPoolDataSource.getPooledConnection(OracleConnectionPoolDataSource.java:114)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getNewPoolOrXAConnection(OracleConnectionCacheImpl.java:694)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getPooledConnection(OracleConnectionCacheImpl.java:540)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:346)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:324)
	at oracle.sysman.util.jdbc.ConnectionCache._getConnection(ConnectionCache.java:296)
	at oracle.sysman.util.jdbc.ConnectionCache.getUnwrappedConnection(ConnectionCache.java:423)
	at oracle.sysman.emSDK.svc.conn.FGAConnectionCache.getFGAConnection(FGAConnectionCache.java:156)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.getPrivateConnection(ConnectionService.java:605)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.getRepositoryVersionAndMode(ConnectionService.java:368)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.verifyRepositoryEx(ConnectionService.java:395)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.verifyRepository(ConnectionService.java:450)
	at oracle.sysman.emdrep.jobs.Dispatcher.mainTask(Dispatcher.java:1952)
	at oracle.sysman.emdrep.omsservice.OmsServiceDriver.run(OmsServiceDriver.java:164)
2013-11-07 02:56:31,446 [OmsServiceDriver thread] ERROR conn.ConnectionService verifyRepositoryEx.433 - Invalid Connection Pool. ERROR = Io 异常: The Network Adapter could not establish the connection
2013-11-07 02:56:56,578 [NotificationMgrThread] WARN  jdbc.ConnectionCache _getConnection.303 - Got a fatal exeption when getting a connection; Error code = 17002; Cleaning up cache and retrying
2013-11-07 02:56:56,578 [NotificationMgrThread] WARN  jdbc.ConnectionCache _getConnection.306 - Io 异常: The Network Adapter could not establish the connection
java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:111)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:145)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:254)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:386)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:493)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:164)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:34)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:767)
	at oracle.jdbc.pool.OracleDataSource.getPhysicalConnection(OracleDataSource.java:324)
	at oracle.jdbc.pool.OracleConnectionPoolDataSource.getPooledConnection(OracleConnectionPoolDataSource.java:114)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getNewPoolOrXAConnection(OracleConnectionCacheImpl.java:694)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getPooledConnection(OracleConnectionCacheImpl.java:540)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:346)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:324)
	at oracle.sysman.util.jdbc.ConnectionCache._getConnection(ConnectionCache.java:296)
	at oracle.sysman.util.jdbc.ConnectionCache.getUnwrappedConnection(ConnectionCache.java:423)
	at oracle.sysman.emSDK.svc.conn.FGAConnectionCache.getFGAConnection(FGAConnectionCache.java:156)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.getSystemConnection(ConnectionService.java:683)
	at oracle.sysman.emdrep.notification.NotificationMgr.getConnection(NotificationMgr.java:252)
	at oracle.sysman.emdrep.notification.NotificationMgr.resetConnection(NotificationMgr.java:298)
	at oracle.sysman.emdrep.notification.NotificationMgr.run(NotificationMgr.java:1190)
2013-11-07 02:56:57,482 [OmsServiceDriver thread] WARN  jdbc.ConnectionCache _getConnection.303 - Got a fatal exeption when getting a connection; Error code = 17002; Cleaning up cache and retrying
2013-11-07 02:56:57,482 [OmsServiceDriver thread] WARN  jdbc.ConnectionCache _getConnection.306 - Io 异常: The Network Adapter could not establish the connection
java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:111)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:145)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:254)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:386)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:493)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:164)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:34)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:767)
	at oracle.jdbc.pool.OracleDataSource.getPhysicalConnection(OracleDataSource.java:324)
	at oracle.jdbc.pool.OracleConnectionPoolDataSource.getPooledConnection(OracleConnectionPoolDataSource.java:114)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getNewPoolOrXAConnection(OracleConnectionCacheImpl.java:694)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getPooledConnection(OracleConnectionCacheImpl.java:540)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:346)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:324)
	at oracle.sysman.util.jdbc.ConnectionCache._getConnection(ConnectionCache.java:296)
	at oracle.sysman.util.jdbc.ConnectionCache.getUnwrappedConnection(ConnectionCache.java:423)
	at oracle.sysman.emSDK.svc.conn.FGAConnectionCache.getFGAConnection(FGAConnectionCache.java:156)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.getSystemConnection(ConnectionService.java:683)
	at oracle.sysman.emdrep.omsservice.OmsServiceDriver.getConnection(OmsServiceDriver.java:88)
	at oracle.sysman.emdrep.jobs.Dispatcher.getConnection(Dispatcher.java:1054)
	at oracle.sysman.emdrep.jobs.Dispatcher.process(Dispatcher.java:1200)
	at oracle.sysman.emdrep.jobs.Dispatcher.mainTask(Dispatcher.java:1912)
	at oracle.sysman.emdrep.omsservice.OmsServiceDriver.run(OmsServiceDriver.java:164)
2013-11-07 02:57:20,789 [XMLLoader0] ERROR emdrep.XMLLoaderContext logLoaderParameters.2179 - 关闭的连接
java.sql.SQLException: 关闭的连接
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:111)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:145)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:207)
	at oracle.jdbc.driver.PhysicalConnection.createStatement(PhysicalConnection.java:912)
	at oracle.jdbc.driver.PhysicalConnection.createStatement(PhysicalConnection.java:888)
	at oracle.jdbc.driver.PhysicalConnection.initUserName(PhysicalConnection.java:3108)
	at oracle.jdbc.driver.PhysicalConnection.getUserName(PhysicalConnection.java:3128)
	at oracle.jdbc.OracleConnectionWrapper.getUserName(OracleConnectionWrapper.java:321)
	at oracle.sysman.util.jdbc.ConnectionWrapper.getUserName(ConnectionWrapper.java:1143)
	at oracle.sql.SQLName.init(SQLName.java:141)
	at oracle.sql.SQLName.<init>(SQLName.java:89)
	at oracle.sql.ArrayDescriptor.createDescriptor(ArrayDescriptor.java:148)
	at oracle.sql.ArrayDescriptor.createDescriptor(ArrayDescriptor.java:114)
	at oracle.sysman.emdrep.dbjava.loader.XMLLoaderContext.logLoaderParameters(XMLLoaderContext.java:2157)
	at oracle.sysman.emdrep.dbjava.loader.XMLLoaderContext.keepAlive(XMLLoaderContext.java:572)
	at oracle.sysman.emdrep.dbjava.loader.XMLLoader.run(XMLLoader.java:1140)
	at java.lang.Thread.run(Thread.java:534)
2013-11-07 02:57:39,525 [OmsServiceDriver thread] WARN  jdbc.ConnectionCache _getConnection.303 - Got a fatal exeption when getting a connection; Error code = 17002; Cleaning up cache and retrying
2013-11-07 02:57:39,525 [OmsServiceDriver thread] WARN  jdbc.ConnectionCache _getConnection.306 - Io 异常: The Network Adapter could not establish the connection
java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:111)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:145)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:254)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:386)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:493)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:164)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:34)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:767)
	at oracle.jdbc.pool.OracleDataSource.getPhysicalConnection(OracleDataSource.java:324)
	at oracle.jdbc.pool.OracleConnectionPoolDataSource.getPooledConnection(OracleConnectionPoolDataSource.java:114)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getNewPoolOrXAConnection(OracleConnectionCacheImpl.java:694)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getPooledConnection(OracleConnectionCacheImpl.java:540)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:346)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:324)
	at oracle.sysman.util.jdbc.ConnectionCache._getConnection(ConnectionCache.java:296)
	at oracle.sysman.util.jdbc.ConnectionCache.getUnwrappedConnection(ConnectionCache.java:423)
	at oracle.sysman.emSDK.svc.conn.FGAConnectionCache.getFGAConnection(FGAConnectionCache.java:156)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.getPrivateConnection(ConnectionService.java:605)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.getRepositoryVersionAndMode(ConnectionService.java:368)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.verifyRepositoryEx(ConnectionService.java:395)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.verifyRepository(ConnectionService.java:450)
	at oracle.sysman.emdrep.jobs.Dispatcher.mainTask(Dispatcher.java:1952)
	at oracle.sysman.emdrep.omsservice.OmsServiceDriver.run(OmsServiceDriver.java:164)
2013-11-07 02:57:43,627 [NotificationMgrThread] WARN  jdbc.ConnectionCache _getConnection.303 - Got a fatal exeption when getting a connection; Error code = 17002; Cleaning up cache and retrying
2013-11-07 02:57:43,627 [NotificationMgrThread] WARN  jdbc.ConnectionCache _getConnection.306 - Io 异常: The Network Adapter could not establish the connection
java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:111)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:145)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:254)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:386)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:493)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:164)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:34)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:767)
	at oracle.jdbc.pool.OracleDataSource.getPhysicalConnection(OracleDataSource.java:324)
	at oracle.jdbc.pool.OracleConnectionPoolDataSource.getPooledConnection(OracleConnectionPoolDataSource.java:114)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getNewPoolOrXAConnection(OracleConnectionCacheImpl.java:694)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getPooledConnection(OracleConnectionCacheImpl.java:540)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:346)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:324)
	at oracle.sysman.util.jdbc.ConnectionCache._getConnection(ConnectionCache.java:296)
	at oracle.sysman.util.jdbc.ConnectionCache.getUnwrappedConnection(ConnectionCache.java:423)
	at oracle.sysman.emSDK.svc.conn.FGAConnectionCache.getFGAConnection(FGAConnectionCache.java:156)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.getSystemConnection(ConnectionService.java:683)
	at oracle.sysman.emdrep.notification.NotificationMgr.getConnection(NotificationMgr.java:252)
	at oracle.sysman.emdrep.notification.NotificationMgr.resetConnection(NotificationMgr.java:298)
	at oracle.sysman.emdrep.notification.NotificationMgr.run(NotificationMgr.java:1190)
2013-11-07 02:58:00,553 [OmsServiceDriver thread] ERROR conn.ConnectionService verifyRepositoryEx.433 - Invalid Connection Pool. ERROR = Io 异常: The Network Adapter could not establish the connection
2013-11-07 02:58:26,605 [OmsServiceDriver thread] WARN  jdbc.ConnectionCache _getConnection.303 - Got a fatal exeption when getting a connection; Error code = 17002; Cleaning up cache and retrying
2013-11-07 02:58:26,605 [OmsServiceDriver thread] WARN  jdbc.ConnectionCache _getConnection.306 - Io 异常: The Network Adapter could not establish the connection
java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:111)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:145)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:254)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:386)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:493)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:164)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:34)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:767)
	at oracle.jdbc.pool.OracleDataSource.getPhysicalConnection(OracleDataSource.java:324)
	at oracle.jdbc.pool.OracleConnectionPoolDataSource.getPooledConnection(OracleConnectionPoolDataSource.java:114)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getNewPoolOrXAConnection(OracleConnectionCacheImpl.java:694)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getPooledConnection(OracleConnectionCacheImpl.java:540)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:346)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:324)
	at oracle.sysman.util.jdbc.ConnectionCache._getConnection(ConnectionCache.java:296)
	at oracle.sysman.util.jdbc.ConnectionCache.getUnwrappedConnection(ConnectionCache.java:423)
	at oracle.sysman.emSDK.svc.conn.FGAConnectionCache.getFGAConnection(FGAConnectionCache.java:156)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.getSystemConnection(ConnectionService.java:683)
	at oracle.sysman.emdrep.omsservice.OmsServiceDriver.getConnection(OmsServiceDriver.java:88)
	at oracle.sysman.emdrep.failover.OMSHeartbeatRecorder.getConnection(OMSHeartbeatRecorder.java:104)
	at oracle.sysman.emdrep.failover.OMSHeartbeatRecorder.heartbeat(OMSHeartbeatRecorder.java:239)
	at oracle.sysman.emdrep.failover.OMSHeartbeatRecorder.mainTask(OMSHeartbeatRecorder.java:427)
	at oracle.sysman.emdrep.omsservice.OmsServiceDriver.run(OmsServiceDriver.java:164)
2013-11-07 02:58:30,646 [NotificationMgrThread] WARN  jdbc.ConnectionCache _getConnection.303 - Got a fatal exeption when getting a connection; Error code = 17002; Cleaning up cache and retrying
2013-11-07 02:58:30,646 [NotificationMgrThread] WARN  jdbc.ConnectionCache _getConnection.306 - Io 异常: The Network Adapter could not establish the connection
java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:111)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:145)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:254)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:386)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:493)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:164)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:34)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:767)
	at oracle.jdbc.pool.OracleDataSource.getPhysicalConnection(OracleDataSource.java:324)
	at oracle.jdbc.pool.OracleConnectionPoolDataSource.getPooledConnection(OracleConnectionPoolDataSource.java:114)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getNewPoolOrXAConnection(OracleConnectionCacheImpl.java:694)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getPooledConnection(OracleConnectionCacheImpl.java:540)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:346)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:324)
	at oracle.sysman.util.jdbc.ConnectionCache._getConnection(ConnectionCache.java:296)
	at oracle.sysman.util.jdbc.ConnectionCache.getUnwrappedConnection(ConnectionCache.java:423)
	at oracle.sysman.emSDK.svc.conn.FGAConnectionCache.getFGAConnection(FGAConnectionCache.java:156)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.getSystemConnection(ConnectionService.java:683)
	at oracle.sysman.emdrep.notification.NotificationMgr.getConnection(NotificationMgr.java:252)
	at oracle.sysman.emdrep.notification.NotificationMgr.resetConnection(NotificationMgr.java:298)
	at oracle.sysman.emdrep.notification.NotificationMgr.run(NotificationMgr.java:1190)
2013-11-07 02:58:47,634 [OmsServiceDriver thread] ERROR em.failover mainTask.431 - java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:111)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:145)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:254)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:386)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:493)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:164)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:34)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:767)
	at oracle.jdbc.pool.OracleDataSource.getPhysicalConnection(OracleDataSource.java:324)
	at oracle.jdbc.pool.OracleConnectionPoolDataSource.getPooledConnection(OracleConnectionPoolDataSource.java:114)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getNewPoolOrXAConnection(OracleConnectionCacheImpl.java:694)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getPooledConnection(OracleConnectionCacheImpl.java:540)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:346)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:324)
	at oracle.sysman.util.jdbc.ConnectionCache._getConnection(ConnectionCache.java:308)
	at oracle.sysman.util.jdbc.ConnectionCache.getUnwrappedConnection(ConnectionCache.java:423)
	at oracle.sysman.emSDK.svc.conn.FGAConnectionCache.getFGAConnection(FGAConnectionCache.java:156)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.getSystemConnection(ConnectionService.java:683)
	at oracle.sysman.emdrep.omsservice.OmsServiceDriver.getConnection(OmsServiceDriver.java:88)
	at oracle.sysman.emdrep.failover.OMSHeartbeatRecorder.getConnection(OMSHeartbeatRecorder.java:104)
	at oracle.sysman.emdrep.failover.OMSHeartbeatRecorder.heartbeat(OMSHeartbeatRecorder.java:239)
	at oracle.sysman.emdrep.failover.OMSHeartbeatRecorder.mainTask(OMSHeartbeatRecorder.java:427)
	at oracle.sysman.emdrep.omsservice.OmsServiceDriver.run(OmsServiceDriver.java:164)
2013-11-07 02:59:08,647 [OmsServiceDriver thread] WARN  jdbc.ConnectionCache _getConnection.303 - Got a fatal exeption when getting a connection; Error code = 17002; Cleaning up cache and retrying
2013-11-07 02:59:08,647 [OmsServiceDriver thread] WARN  jdbc.ConnectionCache _getConnection.306 - Io 异常: The Network Adapter could not establish the connection
java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:111)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:145)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:254)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:386)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:493)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:164)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:34)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:767)
	at oracle.jdbc.pool.OracleDataSource.getPhysicalConnection(OracleDataSource.java:324)
	at oracle.jdbc.pool.OracleConnectionPoolDataSource.getPooledConnection(OracleConnectionPoolDataSource.java:114)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getNewPoolOrXAConnection(OracleConnectionCacheImpl.java:694)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getPooledConnection(OracleConnectionCacheImpl.java:540)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:346)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:324)
	at oracle.sysman.util.jdbc.ConnectionCache._getConnection(ConnectionCache.java:296)
	at oracle.sysman.util.jdbc.ConnectionCache.getUnwrappedConnection(ConnectionCache.java:423)
	at oracle.sysman.emSDK.svc.conn.FGAConnectionCache.getFGAConnection(FGAConnectionCache.java:156)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.getSystemConnection(ConnectionService.java:683)
	at oracle.sysman.emdrep.omsservice.OmsServiceDriver.getConnection(OmsServiceDriver.java:88)
	at oracle.sysman.emdrep.jobs.Dispatcher.getConnection(Dispatcher.java:1054)
	at oracle.sysman.emdrep.jobs.Dispatcher.process(Dispatcher.java:1200)
	at oracle.sysman.emdrep.jobs.Dispatcher.mainTask(Dispatcher.java:1912)
	at oracle.sysman.emdrep.omsservice.OmsServiceDriver.run(OmsServiceDriver.java:164)
2013-11-07 02:59:17,727 [NotificationMgrThread] WARN  jdbc.ConnectionCache _getConnection.303 - Got a fatal exeption when getting a connection; Error code = 17002; Cleaning up cache and retrying
2013-11-07 02:59:17,727 [NotificationMgrThread] WARN  jdbc.ConnectionCache _getConnection.306 - Io 异常: The Network Adapter could not establish the connection
java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:111)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:145)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:254)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:386)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:493)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:164)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:34)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:767)
	at oracle.jdbc.pool.OracleDataSource.getPhysicalConnection(OracleDataSource.java:324)
	at oracle.jdbc.pool.OracleConnectionPoolDataSource.getPooledConnection(OracleConnectionPoolDataSource.java:114)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getNewPoolOrXAConnection(OracleConnectionCacheImpl.java:694)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getPooledConnection(OracleConnectionCacheImpl.java:540)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:346)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:324)
	at oracle.sysman.util.jdbc.ConnectionCache._getConnection(ConnectionCache.java:296)
	at oracle.sysman.util.jdbc.ConnectionCache.getUnwrappedConnection(ConnectionCache.java:423)
	at oracle.sysman.emSDK.svc.conn.FGAConnectionCache.getFGAConnection(FGAConnectionCache.java:156)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.getSystemConnection(ConnectionService.java:683)
	at oracle.sysman.emdrep.notification.NotificationMgr.getConnection(NotificationMgr.java:252)
	at oracle.sysman.emdrep.notification.NotificationMgr.resetConnection(NotificationMgr.java:298)
	at oracle.sysman.emdrep.notification.NotificationMgr.run(NotificationMgr.java:1190)
2013-11-07 02:59:50,736 [OmsServiceDriver thread] WARN  jdbc.ConnectionCache _getConnection.303 - Got a fatal exeption when getting a connection; Error code = 17002; Cleaning up cache and retrying
2013-11-07 02:59:50,736 [OmsServiceDriver thread] WARN  jdbc.ConnectionCache _getConnection.306 - Io 异常: The Network Adapter could not establish the connection
java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:111)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:145)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:254)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:386)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:493)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:164)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:34)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:767)
	at oracle.jdbc.pool.OracleDataSource.getPhysicalConnection(OracleDataSource.java:324)
	at oracle.jdbc.pool.OracleConnectionPoolDataSource.getPooledConnection(OracleConnectionPoolDataSource.java:114)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getNewPoolOrXAConnection(OracleConnectionCacheImpl.java:694)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getPooledConnection(OracleConnectionCacheImpl.java:540)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:346)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:324)
	at oracle.sysman.util.jdbc.ConnectionCache._getConnection(ConnectionCache.java:296)
	at oracle.sysman.util.jdbc.ConnectionCache.getUnwrappedConnection(ConnectionCache.java:423)
	at oracle.sysman.emSDK.svc.conn.FGAConnectionCache.getFGAConnection(FGAConnectionCache.java:156)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.getPrivateConnection(ConnectionService.java:605)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.getRepositoryVersionAndMode(ConnectionService.java:368)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.verifyRepositoryEx(ConnectionService.java:395)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.verifyRepository(ConnectionService.java:450)
	at oracle.sysman.emdrep.jobs.Dispatcher.mainTask(Dispatcher.java:1952)
	at oracle.sysman.emdrep.omsservice.OmsServiceDriver.run(OmsServiceDriver.java:164)
2013-11-07 03:00:04,808 [NotificationMgrThread] WARN  jdbc.ConnectionCache _getConnection.303 - Got a fatal exeption when getting a connection; Error code = 17002; Cleaning up cache and retrying
2013-11-07 03:00:04,808 [NotificationMgrThread] WARN  jdbc.ConnectionCache _getConnection.306 - Io 异常: The Network Adapter could not establish the connection
java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:111)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:145)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:254)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:386)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:493)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:164)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:34)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:767)
	at oracle.jdbc.pool.OracleDataSource.getPhysicalConnection(OracleDataSource.java:324)
	at oracle.jdbc.pool.OracleConnectionPoolDataSource.getPooledConnection(OracleConnectionPoolDataSource.java:114)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getNewPoolOrXAConnection(OracleConnectionCacheImpl.java:694)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getPooledConnection(OracleConnectionCacheImpl.java:540)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:346)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:324)
	at oracle.sysman.util.jdbc.ConnectionCache._getConnection(ConnectionCache.java:296)
	at oracle.sysman.util.jdbc.ConnectionCache.getUnwrappedConnection(ConnectionCache.java:423)
	at oracle.sysman.emSDK.svc.conn.FGAConnectionCache.getFGAConnection(FGAConnectionCache.java:156)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.getSystemConnection(ConnectionService.java:683)
	at oracle.sysman.emdrep.notification.NotificationMgr.getConnection(NotificationMgr.java:252)
	at oracle.sysman.emdrep.notification.NotificationMgr.resetConnection(NotificationMgr.java:298)
	at oracle.sysman.emdrep.notification.NotificationMgr.run(NotificationMgr.java:1190)
2013-11-07 03:00:11,762 [OmsServiceDriver thread] ERROR conn.ConnectionService verifyRepositoryEx.433 - Invalid Connection Pool. ERROR = Io 异常: The Network Adapter could not establish the connection
2013-11-07 03:00:37,798 [OmsServiceDriver thread] WARN  jdbc.ConnectionCache _getConnection.303 - Got a fatal exeption when getting a connection; Error code = 17002; Cleaning up cache and retrying
2013-11-07 03:00:37,798 [OmsServiceDriver thread] WARN  jdbc.ConnectionCache _getConnection.306 - Io 异常: The Network Adapter could not establish the connection
java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:111)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:145)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:254)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:386)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:493)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:164)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:34)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:767)
	at oracle.jdbc.pool.OracleDataSource.getPhysicalConnection(OracleDataSource.java:324)
	at oracle.jdbc.pool.OracleConnectionPoolDataSource.getPooledConnection(OracleConnectionPoolDataSource.java:114)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getNewPoolOrXAConnection(OracleConnectionCacheImpl.java:694)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getPooledConnection(OracleConnectionCacheImpl.java:540)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:346)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:324)
	at oracle.sysman.util.jdbc.ConnectionCache._getConnection(ConnectionCache.java:296)
	at oracle.sysman.util.jdbc.ConnectionCache.getUnwrappedConnection(ConnectionCache.java:423)
	at oracle.sysman.emSDK.svc.conn.FGAConnectionCache.getFGAConnection(FGAConnectionCache.java:156)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.getSystemConnection(ConnectionService.java:683)
	at oracle.sysman.emdrep.omsservice.OmsServiceDriver.getConnection(OmsServiceDriver.java:88)
	at oracle.sysman.emdrep.failover.OMSHeartbeatRecorder.getConnection(OMSHeartbeatRecorder.java:104)
	at oracle.sysman.emdrep.failover.OMSHeartbeatRecorder.heartbeat(OMSHeartbeatRecorder.java:239)
	at oracle.sysman.emdrep.failover.OMSHeartbeatRecorder.mainTask(OMSHeartbeatRecorder.java:427)
	at oracle.sysman.emdrep.omsservice.OmsServiceDriver.run(OmsServiceDriver.java:164)
2013-11-07 03:00:51,844 [NotificationMgrThread] WARN  jdbc.ConnectionCache _getConnection.303 - Got a fatal exeption when getting a connection; Error code = 17002; Cleaning up cache and retrying
2013-11-07 03:00:51,844 [NotificationMgrThread] WARN  jdbc.ConnectionCache _getConnection.306 - Io 异常: The Network Adapter could not establish the connection
java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:111)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:145)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:254)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:386)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:493)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:164)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:34)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:767)
	at oracle.jdbc.pool.OracleDataSource.getPhysicalConnection(OracleDataSource.java:324)
	at oracle.jdbc.pool.OracleConnectionPoolDataSource.getPooledConnection(OracleConnectionPoolDataSource.java:114)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getNewPoolOrXAConnection(OracleConnectionCacheImpl.java:694)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getPooledConnection(OracleConnectionCacheImpl.java:540)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:346)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:324)
	at oracle.sysman.util.jdbc.ConnectionCache._getConnection(ConnectionCache.java:296)
	at oracle.sysman.util.jdbc.ConnectionCache.getUnwrappedConnection(ConnectionCache.java:423)
	at oracle.sysman.emSDK.svc.conn.FGAConnectionCache.getFGAConnection(FGAConnectionCache.java:156)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.getSystemConnection(ConnectionService.java:683)
	at oracle.sysman.emdrep.notification.NotificationMgr.getConnection(NotificationMgr.java:252)
	at oracle.sysman.emdrep.notification.NotificationMgr.resetConnection(NotificationMgr.java:298)
	at oracle.sysman.emdrep.notification.NotificationMgr.run(NotificationMgr.java:1190)
2013-11-07 03:00:58,833 [OmsServiceDriver thread] ERROR em.failover mainTask.431 - java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:111)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:145)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:254)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:386)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:493)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:164)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:34)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:767)
	at oracle.jdbc.pool.OracleDataSource.getPhysicalConnection(OracleDataSource.java:324)
	at oracle.jdbc.pool.OracleConnectionPoolDataSource.getPooledConnection(OracleConnectionPoolDataSource.java:114)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getNewPoolOrXAConnection(OracleConnectionCacheImpl.java:694)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getPooledConnection(OracleConnectionCacheImpl.java:540)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:346)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:324)
	at oracle.sysman.util.jdbc.ConnectionCache._getConnection(ConnectionCache.java:308)
	at oracle.sysman.util.jdbc.ConnectionCache.getUnwrappedConnection(ConnectionCache.java:423)
	at oracle.sysman.emSDK.svc.conn.FGAConnectionCache.getFGAConnection(FGAConnectionCache.java:156)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.getSystemConnection(ConnectionService.java:683)
	at oracle.sysman.emdrep.omsservice.OmsServiceDriver.getConnection(OmsServiceDriver.java:88)
	at oracle.sysman.emdrep.failover.OMSHeartbeatRecorder.getConnection(OMSHeartbeatRecorder.java:104)
	at oracle.sysman.emdrep.failover.OMSHeartbeatRecorder.heartbeat(OMSHeartbeatRecorder.java:239)
	at oracle.sysman.emdrep.failover.OMSHeartbeatRecorder.mainTask(OMSHeartbeatRecorder.java:427)
	at oracle.sysman.emdrep.omsservice.OmsServiceDriver.run(OmsServiceDriver.java:164)
2013-11-07 03:01:19,847 [OmsServiceDriver thread] WARN  jdbc.ConnectionCache _getConnection.303 - Got a fatal exeption when getting a connection; Error code = 17002; Cleaning up cache and retrying
2013-11-07 03:01:19,847 [OmsServiceDriver thread] WARN  jdbc.ConnectionCache _getConnection.306 - Io 异常: The Network Adapter could not establish the connection
java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:111)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:145)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:254)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:386)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:493)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:164)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:34)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:767)
	at oracle.jdbc.pool.OracleDataSource.getPhysicalConnection(OracleDataSource.java:324)
	at oracle.jdbc.pool.OracleConnectionPoolDataSource.getPooledConnection(OracleConnectionPoolDataSource.java:114)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getNewPoolOrXAConnection(OracleConnectionCacheImpl.java:694)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getPooledConnection(OracleConnectionCacheImpl.java:540)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:346)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:324)
	at oracle.sysman.util.jdbc.ConnectionCache._getConnection(ConnectionCache.java:296)
	at oracle.sysman.util.jdbc.ConnectionCache.getUnwrappedConnection(ConnectionCache.java:423)
	at oracle.sysman.emSDK.svc.conn.FGAConnectionCache.getFGAConnection(FGAConnectionCache.java:156)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.getSystemConnection(ConnectionService.java:683)
	at oracle.sysman.emdrep.omsservice.OmsServiceDriver.getConnection(OmsServiceDriver.java:88)
	at oracle.sysman.emdrep.jobs.Dispatcher.getConnection(Dispatcher.java:1054)
	at oracle.sysman.emdrep.jobs.Dispatcher.process(Dispatcher.java:1200)
	at oracle.sysman.emdrep.jobs.Dispatcher.mainTask(Dispatcher.java:1912)
	at oracle.sysman.emdrep.omsservice.OmsServiceDriver.run(OmsServiceDriver.java:164)
2013-11-07 03:01:38,880 [NotificationMgrThread] WARN  jdbc.ConnectionCache _getConnection.303 - Got a fatal exeption when getting a connection; Error code = 17002; Cleaning up cache and retrying
2013-11-07 03:01:38,880 [NotificationMgrThread] WARN  jdbc.ConnectionCache _getConnection.306 - Io 异常: The Network Adapter could not establish the connection
java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:111)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:145)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:254)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:386)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:493)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:164)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:34)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:767)
	at oracle.jdbc.pool.OracleDataSource.getPhysicalConnection(OracleDataSource.java:324)
	at oracle.jdbc.pool.OracleConnectionPoolDataSource.getPooledConnection(OracleConnectionPoolDataSource.java:114)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getNewPoolOrXAConnection(OracleConnectionCacheImpl.java:694)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getPooledConnection(OracleConnectionCacheImpl.java:540)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:346)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:324)
	at oracle.sysman.util.jdbc.ConnectionCache._getConnection(ConnectionCache.java:296)
	at oracle.sysman.util.jdbc.ConnectionCache.getUnwrappedConnection(ConnectionCache.java:423)
	at oracle.sysman.emSDK.svc.conn.FGAConnectionCache.getFGAConnection(FGAConnectionCache.java:156)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.getSystemConnection(ConnectionService.java:683)
	at oracle.sysman.emdrep.notification.NotificationMgr.getConnection(NotificationMgr.java:252)
	at oracle.sysman.emdrep.notification.NotificationMgr.resetConnection(NotificationMgr.java:298)
	at oracle.sysman.emdrep.notification.NotificationMgr.run(NotificationMgr.java:1190)
2013-11-07 03:02:01,891 [OmsServiceDriver thread] WARN  jdbc.ConnectionCache _getConnection.303 - Got a fatal exeption when getting a connection; Error code = 17002; Cleaning up cache and retrying
2013-11-07 03:02:01,891 [OmsServiceDriver thread] WARN  jdbc.ConnectionCache _getConnection.306 - Io 异常: The Network Adapter could not establish the connection
java.sql.SQLException: Io 异常: The Network Adapter could not establish the connection
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:111)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:145)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:254)
	at oracle.jdbc.driver.T4CConnection.logon(T4CConnection.java:386)
	at oracle.jdbc.driver.PhysicalConnection.<init>(PhysicalConnection.java:493)
	at oracle.jdbc.driver.T4CConnection.<init>(T4CConnection.java:164)
	at oracle.jdbc.driver.T4CDriverExtension.getConnection(T4CDriverExtension.java:34)
	at oracle.jdbc.driver.OracleDriver.connect(OracleDriver.java:767)
	at oracle.jdbc.pool.OracleDataSource.getPhysicalConnection(OracleDataSource.java:324)
	at oracle.jdbc.pool.OracleConnectionPoolDataSource.getPooledConnection(OracleConnectionPoolDataSource.java:114)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getNewPoolOrXAConnection(OracleConnectionCacheImpl.java:694)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getPooledConnection(OracleConnectionCacheImpl.java:540)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:346)
	at oracle.jdbc.pool.OracleConnectionCacheImpl.getConnection(OracleConnectionCacheImpl.java:324)
	at oracle.sysman.util.jdbc.ConnectionCache._getConnection(ConnectionCache.java:296)
	at oracle.sysman.util.jdbc.ConnectionCache.getUnwrappedConnection(ConnectionCache.java:423)
	at oracle.sysman.emSDK.svc.conn.FGAConnectionCache.getFGAConnection(FGAConnectionCache.java:156)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.getPrivateConnection(ConnectionService.java:605)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.getRepositoryVersionAndMode(ConnectionService.java:368)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.verifyRepositoryEx(ConnectionService.java:395)
	at oracle.sysman.emSDK.svc.conn.ConnectionService.verifyRepository(ConnectionService.java:450)
	at oracle.sysman.emdrep.jobs.Dispatcher.mainTask(Dispatcher.java:1952)
	at oracle.sysman.emdrep.omsservice.OmsServiceDriver.run(OmsServiceDriver.java:164)
2013-11-07 03:02:21,213 [XMLLoader0] ERROR emdrep.XMLLoaderContext logLoaderParameters.2179 - 关闭的连接
java.sql.SQLException: 关闭的连接
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:111)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:145)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:207)
	at oracle.jdbc.driver.PhysicalConnection.createStatement(PhysicalConnection.java:912)
	at oracle.jdbc.driver.PhysicalConnection.createStatement(PhysicalConnection.java:888)
	at oracle.jdbc.driver.PhysicalConnection.initUserName(PhysicalConnection.java:3108)
	at oracle.jdbc.driver.PhysicalConnection.getUserName(PhysicalConnection.java:3128)
	at oracle.jdbc.OracleConnectionWrapper.getUserName(OracleConnectionWrapper.java:321)
	at oracle.sysman.util.jdbc.ConnectionWrapper.getUserName(ConnectionWrapper.java:1143)
	at oracle.sql.SQLName.init(SQLName.java:141)
	at oracle.sql.SQLName.<init>(SQLName.java:89)
	at oracle.sql.ArrayDescriptor.createDescriptor(ArrayDescriptor.java:148)
	at oracle.sql.ArrayDescriptor.createDescriptor(ArrayDescriptor.java:114)
	at oracle.sysman.emdrep.dbjava.loader.XMLLoaderContext.logLoaderParameters(XMLLoaderContext.java:2157)
	at oracle.sysman.emdrep.dbjava.loader.XMLLoaderContext.keepAlive(XMLLoaderContext.java:572)
	at oracle.sysman.emdrep.dbjava.loader.XMLLoader.run(XMLLoader.java:1140)
	at java.lang.Thread.run(Thread.java:534)
2013-11-07 03:02:22,898 [OmsServiceDriver thread] ERROR conn.ConnectionService verifyRepositoryEx.433 - Invalid Connection Pool. ERROR = Io 异常: The Network Adapter could not establish the connection
2013-11-07 03:02:22,898 [OmsServiceDriver thread] ERROR em.jobs die.1129 - Dispatcher: causing a restart
2013-11-07 03:02:22,898 [OmsServiceDriver thread] ERROR emd.main restart.389 - HealthMonitor 2013-11-7 3:02:22 Job Dispatcher error: Could not get connection

